### DISCUSSION TITLE
Content item admin summary display

### DISCUSSION THREAD

**Discussion by agriffard** [OrchardCMS/OrchardCore#14383] [Q&A] [answered]
Can someone please remember me how you can specify for a content type that you want to display some additional informations in the Admin Summary.

Example: You have an Excerpt HtmlField on a BlogPost and you want to make it appear in the admin content items list.

Probably an Admin template with a specific (`..._SummaryAdmin`) name but I don't find docs or examples.

**Comment by MikeAlhayek** [comment]
@agriffard  You can do that using placement like this

```json
{
    "HtmlField": [
    {
           "displayType": "SummaryAdmin",
           "differentiator": "YourContentPart-Data",
           "place": "Content"
    }
   ]
}
```

Assuming you have this part

```C#
public class YourContentPart : ContentPart
{
    public HtmlField Data { get; set; }
}

**Reply by agriffard** [reply]
So I need a custom admin theme?
It doesn't seem to work, as reported here: https://github.com/OrchardCMS/OrchardCore/issues/2935.

**Reply by agriffard** [reply]
Can I target the `Subtitle` TextField of a BlogPost (with TheBlogTheme recipe) using Admin Placement?

**Reply by MikeAlhayek** [reply, marked as answer]
@agriffard you don't need to have a custom theme. You just need to define placement either by adding `placement.json` file in your project or using the "Placements" feature which allow you to define custom placement using the UI

To target `Subtitle` using the "Placements" feature. Use the UI to add a new shape called `TextField` with the following JSON

```json
[
    {
           "displayType": "SummaryAdmin",
           "differentiator": "BlogPost-Subtitle",
           "place": "Content"
    }
]
```
or if you user `placement.json`, then add the following
```json
{
    "TextField": [
    {
           "displayType": "SummaryAdmin",
           "differentiator": "BlogPost-Subtitle",
           "place": "Content"
    }
   ]
}
```

I have no texted the above, but should work

**Comment by agriffard** [comment]
![image](https://github.com/OrchardCMS/OrchardCore/assets/703248/2cf521f9-5c52-4ad3-930f-78edae59a541)

![image](https://github.com/OrchardCMS/OrchardCore/assets/703248/de76bf8a-ba92-4705-8cd6-927224eeccf1)
